integration_tests/macros/e2e_tests/tests_validation.sql

```diff
@@ -119,8 +119,7 @@
             {% do dimensions_with_problems.append[dimensions] %}
         {% endif %}
     {% endfor %}
-
-    {% if results | length != 2 %}
+    {% if results | length != 3 %}
```

Comment:
I'm not sure why this is hardcoded in the first place, but I added another failed test so this is the new wanted value

Reply:
Yes I hate that, I did this on the weekend to start moving in a better direction:
https://github.com/elementary-data/dbt-data-reliability/pull/313
(Validate that we only have errors in tests that have an `error_test` tag)